v3/util/authorizedkeys_command.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENCRYPTION_ID=$(etcdctl get /klam-ssh/ENCRYPTION_ID)
+ENCRYPTION_KEY=$(etcdctl get /klam-ssh/ENCRYPTION_KEY)
+ROLE_NAME=$(etcdctl get /klam-ssh/ROLE_NAME)
+KEY_LOCATION_PREFIX=$(etcdctl get /klam-ssh/KEY_LOCATION_PREFIX)
+IMAGE=$(etcdctl get /images/klam-ssh)
```

Comment:
Ah, I see your point.  I was relying on the configs from the infrastructure repo to do this instead.  I've set a default value in v3/setup/klam-ssh.sh